src/main/java/com/box/sdk/LargeFileUpload.java

```diff
@@ -165,6 +165,8 @@ public LargeFileUpload() {
         long offset = 0;
         long processed = 0;
         int partPostion = 0;
+        //Set the Max Queue Size to 1.5x the number of processors
+        int maxQueueSize = Double.valueOf(this.executorService.getMaximumPoolSize() * 1.5).intValue();
```

Comment:
Could we do `Math.ceil()` or `Math.floor()` here for clarity?

Reply:
Added the explicit call to Math.ceil() to explicitly round up.